com.endlessm.EknServices.json.in

```diff
@@ -9,6 +9,7 @@
         "--filesystem=/var/endless-extra/flatpak:ro",
         "--filesystem=~/.local/share/flatpak:ro",
         "--filesystem=~/.local/share",
+        "--filesystem=home",
```

Comment:
I don't think this is necessary. The line above it already gives you write access to the place where you need to symlink the shards.

Reply:
That's true. I think this is a leftover anyway.